cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -21,35 +21,59 @@ import { IWidgetProps } from 'components/AbstractWidget';
 import { objectQuery } from 'services/helpers';
 import { WIDGET_PROPTYPES } from 'components/AbstractWidget/constants';
 import withStyles from '@material-ui/core/styles/withStyles';
-
+import { blue } from 'components/ThemeWrapper/colors';
```

Comment:
Technically we should be able to use Theme since at some point we wrap this in `ThemeWrapper` for the theme to be passed down the context. But our definition of colors in the theme is not right. `Palette` in material-ui is not defined to have colors the way we define.

Since I am still using the color used in the theme file, and even if we decide to change the color this directly gets affected. For now I would suggest let us have this. We need to fix the way we define colors in material-ui so that we could use the color we define directly from the theme without any hacks. Right now if I have to use the blue from the theme I have to type alias it to any (`outline: '1px solid ${(theme.palette as any).blue[100]}'`)